site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -4,145 +4,201 @@ name: Interest Invokers (Explainer)
+
+1. When the long-pressed element has `interesttarget`, any UA-provided context menu will not blur or otherwise obscure the web content it covers. In this way, the page will still be visible "under" the context menu.
+2. The portion of the renderer window that is still visible (not obscured by the context menu) will be exposed to the developer via four new [`env()` values](https://developer.mozilla.org/en-US/docs/Web/CSS/env), tentatively called `context-menu-inset-top`, `context-menu-inset-right`, `context-menu-inset-bottom`, and `context-menu-inset-left`. In this way, the developer can safely position the target popover within the area that isn't obscured by the context menu. (See [the mockups section](#mockups).)
+3. An attempt will be made to provide at least half of the screen (or a "reasonable" amount of screen real estate) for developer content, e.g. by moving the context menu down rather than centering it within the viewport.
+
+See https://github.com/openui/open-ui/issues/1052 for a much more detailed conversation with developers about touch screen behavior. That extended discussion led to the behaviors described in this section.
+
+### Other
+
+Th "other" category includes many HID types, including assistive technologies, eye tracking interfaces (e.g. [Apple Vision Pro](https://www.apple.com/apple-vision-pro/)), voice control systems, switch access, and future HIDs that haven't been developed yet.
+
+Since this category of HIDs contains less-commonly-used HID types, and because it also contains future HIDs that don't even exist yet, the specification for `interesttarget` will simply *mandate* that the User Agent provide *a way* for users to both "show interest" and "lose interest" in elements, no matter what HID is in use, without specifying exactly what form that might take. In some cases, this behavior likely cannot be specified, and must be left to the discretion of the user agent. For example, an explicit hand gesture might be needed for the Vision Pro interface. In the to-be-invented case, it stands to reason that devices might introduce new and novel concepts of showing/losing interest.
+
+## Mouse delays
+
+When the HID is a mouse, and hover/de-hover is the method for showing and losing interest, delays are important for a number of reasons:
```

Comment:
nit: I think the first and should be dropped, it introduces ambiguity. I read this to mean that you can have the HID be a mouse and the method is something other than hover.

Reply:
Agreed. I changed "and" to "where". LMK if that feels more clear.